profile/rwahs.xml

```diff
@@ -7924,6 +7947,113 @@
         </placement>
       </bundlePlacements>
     </display>
+    <display code="photograph_search_display" type="ca_objects" system="1">
+      <labels>
+        <label locale="en_AU">
+          <name>Photograph Object search</name>
+        </label>
+      </labels>
+      <typeRestrictions>
+        <restriction type="photograph"/>
+      </typeRestrictions>
+      <groupAccess>
+        <permission group="admin" access="edit"/>
+        <permission group="library" access="read"/>
+        <permission group="museum" access="read"/>
+        <permission group="photograph" access="read"/>
+        <permission group="memorial" access="read"/>
+      </groupAccess>
+      <bundlePlacements>
+        <placement code="ca_objects_preferred_labels">
+          <bundle>ca_objects.preferred_labels</bundle>
+          <settings>
+            <setting name="label" locale="en_AU">Title</setting>
+            <setting name="format"><![CDATA[<div style="min-width:30em;">^preferred_labels</div>]]></setting>
```

Comment:
I can see this has been copied from the existing displays, but should these settings, in particular `format`, be here?  Wouldn't it give what we want if only the `label` and `maximum_width` settings were given here?